### ISSUE TITLE
Issue with Peers Display on Kick/Kick Ban/Kick Ban For

### ISSUE THREAD

**Issue by NaPurHab** [BiglySoftware/BiglyBT#733]
When using the Kick/Kick Ban/Kick Ban For, the Peers list disappears. If I select a different torrent, then come back the Peer List is populated. But do any of the Kick commands and the list blanks out.

This only occurs in the Active View when using the Popup Views from the bottom (General, Sources, Peers, Swarm, Etc.). Click Torrent, then Click Peers at bottom Of screen, select connection to Kick, then right click and choose which Kick to use. Peer View will then blank.

I have tried but can only reproduce this in the Active View.

* OS and version: Windows 10 1803
* BiglyBT Version Number: 1.7.0.1_B20

**Comment by parg** [comment]
Does this happen reliably for you? I managed to reproduce it once or twice but not in general. Does the peers view go grey/disabled or stay white but empty?

**Comment by NaPurHab** [comment]
I have the color scheme set to have white/mint stripes. So when it happens I see the strips with no text. It just blanks the peers view.
As to being reliable, it is more than not. Say about 2/3 of the time.
Just started a torrent and when I tried to kick it blanked, I de-selected the torrent, then selected, still blank, de-selected a 2nd time, then re-slelected and the peers list was back. I was able to recreated this 3 times just now.
All I can say is you have been able to reproduce it, so it exists. But since it doesn't happen all the time, it is going to be tricky to track down.
